paper/paper.md

```diff
@@ -30,40 +30,40 @@ bibliography: paper.bib
 
 ![Number of user since first version](easyaccess_users.png)
 
 # `easyaccess`
 
-`easyaccess` is a command line interpreter which is heavily based in the [`cmd`](https://docs.python.org/3/library/cmd.html) Python core module and `termcolor` [@termcolor], at its core as well other external and open sourced libraries including NumPy [@NumPy], `pandas` [@pandas], `fitsio` [@fitsio] and `h5py` [@h5py] to handle and transform array data coming to or from the DB, `cx_Oracle` [@cxoracle] to handle the Oracle communication and `requests` [@requests] for external URL requests. Figure 2 shows an example of
-the welcome screen as seen as a DES user.
+`easyaccess` is a command line interpreter that is heavily based on `termcolor` [@termcolor] and the [`cmd`](https://docs.python.org/3/library/cmd.html) Python core module. It interfaces with `cx_Oracle` [@cxoracle] to communicate with Oracle, `requests` [@requests] for external URL requests, and other external open source libraries, including NumPy [@NumPy], `pandas` [@pandas], `fitsio` [@fitsio] and `h5py` [@h5py] to handle and transform array data. 
+Figure 2 shows an example of the welcome screen as seen by a DES user.
 
 ![Welcome screenshot](easyaccess_welcome.png)
 
 ## Features
 
- `easyaccess` has a variety of features including a history of past commands and smart tab  auto-completion for commands, functions, columns, users, tables, and paths as being typed. Tables can be written directly into comma-separated-values (CSV) files (or white-space separated), FITS [@FITS], and HDF5 [@hdf5] files and an iterator is provided to avoid memory constraints when retrieving large tables. Tables can also be displayed on the screen and most of the formatting is done using `pandas`. Similarly, users with DB space can easily upload tables from any of the file format described above and share with other users. The uploading mechanism is done chunk-wise, allowing large tables to be loaded while keeping memory usage low.
+ `easyaccess` has a variety of features including a history of past commands and smart tab auto-completion for commands, functions, columns, users, tables, and paths. Tables can be written directly into comma-separated-value (CSV) or white-space separated text files, FITS [@FITS] files, and HDF5 [@hdf5] files. It provides an iteratation scheme to avoid memory constraints when retrieving large tables. Tables can also be displayed on the command line and most of the formatting is done using `pandas`. Similarly, priviledged users can easily upload tables to the database from any of the file format described above in order to share data with other users. The uploading mechanism is done chunk-wise, allowing large tables to be loaded while keeping memory usage low.
```

Comment:
I'm not really sure what was meant by "an iterator is provided", so take a look at my changes.